ask/commands.py

```diff
@@ -0,0 +1,182 @@
+import sublime
+import sublime_plugin
+import pieces_os_client as pos_client
+from pieces.settings import PiecesSettings
+import re
+from difflib import Differ 
+from .prompts import *
+import mdpopups
+import time
+LIST_ITEMS = [
+	("Fix a bug",BUGS_PROMPT),
+	("Find a possible bug", POSSIBLE_BUGS_PROMPT),
+	("Write cleaner code", CLEANER_CODE_PROMPT),
+	("Add a doc string to this function",DOC_STRING_PROMPT),
+	("Add comments to the code",ADD_COMMENTS_PROMPT)
```

Comment:
we should use the prompt task pipelines for some of these, do you know how to use them?

Reply:
+1 here I would add a Github issue for this